packages/api/pages/api/users/[userId].ts

```diff
@@ -53,6 +61,60 @@ export default createRoute<{ userId: string }>()
         });
       }
 
+      if (name) {
+        await client.authUser.update({
+          where: {
+            id: req.query.userId,
+          },
+          data: { name },
+        });
+      }
+
+      if (password || email) {
+        const key = (await auth.getAllUserKeys(user.id)).find(
+          (key) => key.providerId === 'username'
+        );
+        if (key) {
```

Comment:
This makes sure that only the actual user can change their  password or email, correct?

Reply:
If so, an explanatory comment could be helpful.

Reply:
No, this is how we find the key that contains their password so we can update it or check that the email changed before we start the verification process